provision.azure.yaml

```diff
@@ -1,3 +1,5 @@
 resources:
   topics:
-    - name: alert
+    - name: ffc-pay-alerts
+  storage:
+    - name: ffcpayalerts
```

Comment:
Required as the pipeline failed on the `Provision function app` stage (https://jenkins-ffc.azure.defra.cloud/blue/organizations/jenkins/ffc-pay%2Fffc-pay-alerts-build/detail/main/1/pipeline).

Looking into Groovy scripts it requires a `storage` resource (https://github.com/DEFRA/ffc-jenkins-pipeline-library/blob/39c1b6395115511d14b006f9b14d8168de0535e2/src/uk/gov/defra/ffc/Function.groovy#L22)